python/nav/etc/webfront/webfront.conf

```diff
@@ -110,6 +110,10 @@ enabled = no
 # authenticated user?
 #varname = REMOTE_USER
 
+# Whether usernames set in REMOTE_USER should automatically be created in the
+# database if they do not already exist.
```

Comment:
Just a question about phrasing: for each login there is only one username set in REMOTE_USER, right? Because to me this makes it sound like it could be multiple

Reply:
True